components/mosart/src/riverroute/MOSART_stra_mod.F90

```diff
@@ -41,7 +41,7 @@ subroutine stratification(iunit, localDeltaT,nt)
 		real(r8) :: le= 2.501e6													! latent heat of vaporaization (J/kg)
 		real(r8) :: rho_w = 1.e3												! Water density  (kg/m3)
 		real(r8) :: rho_a = 1.177_r8											! Air density  (kg/m3)
-		real(r8) :: st_bl = 5.67e-8												! Stefan-Boltzmann constant ~ W/m^2/K^4
+		real(r8) :: st_bl = 5.670374419e-8												! Stefan-Boltzmann constant ~ W/m^2/K^4
```

Comment:
Instead of setting the value explicitly, use the shr_const_mod module and set it to SHR_CONST_STEBOL.

Reply:
I believe the same applies here, but a MOSART person would need to check that this is needed for stand-alone.